### PR TITLE
refractor: rename layer0 to edgio

### PR DESCRIPTION
**Pull request by rishi-raj-jain**

### CODE REVIEW

File: playground/nuxt.config.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ export default defineNuxtConfig({
       baseURL: 'https://netlify-photo-gallery.netlify.app'
     },
     layer0: {},
```
Comment (pi0) [inline playground/nuxt.config.ts:55]: Shall we remove older layer0?

File: docs/content/5.providers/edgio.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Edgio
+
+Optimize images with Edgio's optimization service
```
Comment (pi0) [inline docs/content/5.providers/edgio.md:3]: Maybe add something like formerly layer 0 to reduce confusion and improve SEO?

File: src/types/module.ts [hunk 3]
```diff
@@ -51,6 +51,7 @@ export interface ImageProviders {
   imagekit?: any
   imgix?: any
   layer0?: any
```
Comment (pi0) [inline src/types/module.ts:53]: this should be removed too

File: src/provider.ts [hunk 4]
```diff
@@ -21,6 +21,7 @@ const BuiltInProviders = [
   'ipx',
   'netlify',
   'layer0',
```
Comment (pi0) [inline src/provider.ts:23]: we removed provider...

**Review by pi0 — COMMENTED** [review]
Thanks. I've added some suggestions mainly to either drop `layer0` from code or make it an alias provider to edgio

File: playground/nuxt.config.ts (hunk 1 above)
Comment (rishi-raj-jain) [reply playground/nuxt.config.ts:55]: I think there was no baseURL specified before, shall I update both the configs to have one?

Comment (pi0) [reply playground/nuxt.config.ts:55]: Good idea

File: src/provider.ts (hunk 4 above)
Comment (rishi-raj-jain) [reply src/provider.ts:23]: Wouldn't that be breaking one?

Comment (pi0) [reply src/provider.ts:23]: Image v1 is not officially released yet but of course we can also support it as alias if you like...

**Review by pi0 — APPROVED** [review]

**Comment by rishi-raj-jain** [comment]
@pi0 Any idea why that docus based redirect is breaking?

**Pull request merged**